lib/email_messages.py

```diff
@@ -61,3 +62,28 @@ def email_course_error(request, exercise, message, exception=True):
         email_course_instance(instance, subject, body)
     except:
         pass
+
+
+def email_course_students(
+        instance: CourseInstance,
+        subject: str,
+        message: str,
+        audience: CourseInstance.ENROLLMENT_AUDIENCE=CourseInstance.ENROLLMENT_AUDIENCE.ALL_USERS
+        ) -> bool:
+    """
+    Sends an email to students on the course. Audience parameter controls whether the mail goes
+    to all (default), just internal, or just external students.
+    """
+    students = instance.students
+    if audience == instance.ENROLLMENT_AUDIENCE.INTERNAL_USERS:
+        students = students.filter(organization=settings.LOCAL_ORGANIZATION)
+    if audience == instance.ENROLLMENT_AUDIENCE.EXTERNAL_USERS:
```

Comment:
```suggestion
    elif audience == instance.ENROLLMENT_AUDIENCE.EXTERNAL_USERS:
```
Both if conditions can not be True in this case.